railway/railway.md

```diff
@@ -0,0 +1,81 @@
+---
+Documentation for Railway Template for Coder
+https://railway.app/template/cUQ8_P?referralCode=tfH8Uw
+---
+
+<div align="center">
+  <a href="https://coder.com#gh-dark-mode-only">
+    <img src="https://raw.githubusercontent.com/coder/coder/main/docs/images/logo-white.png" style="width: 128px">
+  </a>
+  <h3>
+  Self-Hosted Remote Development Environments
+  </h3>
+  <a href="https://coder.com">
+    <img style="width: 650px" src="https://raw.githubusercontent.com/coder/coder/main/docs/images/banner-white.png">
+  </a>
+</div>
```

Comment:
Let's remove this. Looks too marketing-y

Reply:
ok. I thought we may do some marketing.